lib/rules/no-multiple-template-root.js

```diff
@@ -37,6 +102,20 @@ module.exports = {
           return
         }
 
+        const comments = element.comments
+        if (disallowComments && comments.length > 0) {
+          const needReportComments = getReportComments(comments, element)
+          if (needReportComments.length > 0) {
+            for (const comment of needReportComments) {
+              context.report({
+                node: comment,
+                loc: comment.loc,
+                messageId: 'commentRoot'
+              })
+            }
+          }
```

Comment:
I don't think the `if` is necessary.

```suggestion
          for (const comment of getReportComments(comments, element)) {
            context.report({
              node: comment,
              loc: comment.loc,
              messageId: 'commentRoot'
            })
          }
```